variables.tf

```
@@ -1 +1,172 @@
+### REQUIRED Variables
+variable "name" {
+  description = "Name of the vault stack, will be use to prefix resources"
+  type        = "string"
+}
 
+variable "environment" {
+  description = "Type of environment -- must be one of: dev, test, prod"
+  type        = "string"
+}
+
+variable "ami_owner" {
+  description = "Account id/alias of the AMI owner"
+  type        = "string"
+}
+
+variable "key_pair_name" {
+  description = "Keypair to associate to launched instances"
+  type        = "string"
+}
+
+variable "additional_ips_allow_inbound" {
+  description = "List of ip address that allow to have access to resources"
+  type        = "list"
+}
+
+variable "ec2_extra_security_group_ids" {
+  description = "List of additional security groups to add to EC2 instances"
+  type        = "list"
+}
+
+variable "ec2_subnet_ids" {
+  description = "List of subnets where EC2 instances will be launched"
+  type        = "list"
+}
+
+variable "lb_certificate_arn" {
+  type        = "string"
+  description = "Arn of a created certificate to be use for the load balancer"
+}
+
+variable "lb_subnet_ids" {
+  description = "List of subnets to associate to the Load Balancer"
+  type        = "list"
+}
+
+variable "vault_version" {
+  description = "Version of Vault to be installed on servers"
+  type        = "string"
+}
+
+variable "vault_dns_address" {
+  type        = "string"
+  description = "The DNS address that vault will be accessible at"
+}
+
+variable "kms_key_id" {
+  description = "Id of an AWS KMS key use for auto unseal operation when vault is intialize"
+  type        = "string"
+}
+
+variable "dynamodb_table" {
+  description = "Name of the Dynamodb to be used as storage backend for Vault"
+  type        = "string"
+}
+
+variable "zone_id" {
```

is this a route53 zone id? 
```suggestion
variable "route53_zone_id" {
```